pages/committees.tsx

```diff
@@ -7,22 +7,39 @@ const Committees = () => {
   const [isLoading, setIsLoading] = useState(true);
   const [committees, setCommittees] = useState<owCommitteeType[]>([]);
 
+  const excludedCommittees = [
+    "Jubkom",
+    "Velkom",
+    "Output",
+    "Ekskom",
+    "Faddere",
+    "Debug",
```

Comment:
Hvis denne siden skal funke som en ish infoside om alle komiteene man kan søke på, synes jeg ikke at vi skal ekskludere velkom, ekskom og debug

Reply:
Ser den. Tenker det bare er litt misvisende å vise dem når man ikke kan søke dem. Hva med en badge som det står "Har opptak" ellerno. For de komiteene som har opptak?

Reply:
Ouuu, ja det kunne funka

Reply:
La det til nå, gjerne tweak litt på stylingen

Reply:
ser ikke ut som at det funker?